atk/src/attribute.rs

```diff
@@ -1,6 +1,4 @@
-// Copyright 2013-2018, The Gtk-rs Project Developers.
-// See the COPYRIGHT file at the top-level directory of this distribution.
-// Licensed under the MIT license, see the LICENSE file or <https://opensource.org/licenses/MIT>
+// Take a look at the license at the top of the repository in the LICENSE file.
```

Comment:
That should actually be at the top of the crate in addition too, so that it ends up in what gets uploaded to crates.io.

The text can stay the same here, but we should link the license file into each crate's root.